.eslintrc.json

```diff
@@ -58,7 +58,8 @@
         "@typescript-eslint/comma-dangle": ["error", "only-multiline"],
         "@typescript-eslint/explicit-module-boundary-types": "off",
         "@typescript-eslint/ban-ts-comment": "off",
-        "@typescript-eslint/ban-types": "off"
+        "@typescript-eslint/ban-types": "off",
+        "import/no-extraneous-dependencies": "off"
```

Comment:
tell me please why we need `import/no-extraneous-dependencies: off`?

Reply:
Actually we need to use this versions:
"react": "18.2.0",
"react-dom": "18.2.0",
But the problem is  "react-virtualized": "~9.22.0"  get error react@"^15.3.0 || ^16.0.0-alpha"

https://github.com/bvaughn/react-virtualized/issues/1746